crates/re_sdk/src/msg_sender.rs

```diff
@@ -320,6 +320,7 @@ impl MsgSender {
 
     /// Consumes, packs, sanity checks and finally sends the message to the currently configured
     /// target of the SDK.
+    #[allow(clippy::unnecessary_wraps)] // TODO(emilk): remove the Result?
```

Comment:
I wouldn't; this can easily become fallible in the future, at which point introducing a result type would require some nasty refactoring on users' end.

Much easier to design with fallibility in mind from the beginning imo.